packages/visual-dev/src/components/GlobalStyle/GlobalStyle.tsx

```diff
@@ -38,32 +38,36 @@ export const CSSVariables = createGlobalStyle`
   --sq-action-secondary-border: #a6b9bd;
   --sq-text-on-secondary: #575757;
   --sq-text-on-secondary-hovered: #ffffff;
-  --sq-placeholder-text-on-secondary: #bdbdbd;
+  --sq-placeholder-text-on-secondary: #aaaaaa;
 
   /* Success */
   --sq-surface-success: #57ac59;
   --sq-surface-success-hovered: #479449;
   --sq-surface-success-subdued: #e4fce3;
   --sq-border-success: #57ac59;
+  --sq-badge-surface-success: #e6faf4;
+  --sq-badge-text-success: #146352;
 
   /* Warning */
   --sq-surface-warning: #f1c359;
   --sq-surface-warning-subdued: #fcf8e3;
   --sq-border-warning: #f1c359;
+  --sq-badge-surface-warning: #ffe3a1;
```

Comment:
Doesn't match Figma hex value

Reply:
Confirmed, I believe this is a big in Figma (the swatch colour doesn't match the hex values, preferring the swatch colour).